docs/dietpi_tools.md

```diff
@@ -2,142 +2,170 @@
 
 ## DietPi-Launcher
 
+**dietpi-software** will be automatically ran on the first login. It could be accessed at any time running next next from the command line:
+
+``` bash
+dietpi-launcher
+```
+
 It is the main program to run whenever you have a DietPi OS. It provides a quick access to all the main DietPi programs.
 
 ![dietpi-launcher](assets/images/dietpi-launcher.jpg)
 
 ## DietPi-Software
 
-DietPi-Software will be automatically displayed on the first login. It is one of the core tools, enabling the installation of one or multiple software items, either:
+**dietpi-software** will be automatically displayed on the first login. It could be accessed at any time running next next from the command line:
+
+``` bash
+dietpi-software
+```
+
+![dietpi-software](assets/images/dietpi-software.jpg)
+
+It is one of the core tools, enabling you to install or uninstall one or more 
+[**DietPi optimised software**](../software/) titles.
```

Comment:
I think this link should be ../dietpi_optimised_software.

Reply:
Wouldn't it be reasonable to have the software overview page as `/software/index.html`?
Because currently that path has no index page which leads to: https://dietpi.com/docs/software/
This would mean that we need to rename the file to `software.md` 🤔. Not sure if best, but otherwise for now as Stephan said 😉.

Reply:
I agree with Stephan suggestion. Let's go with it.
Locally it rendered fine, but indeed on dietpi.com is not ok. I'll add more details in the tasks doc for the future ideas..